src/includes/enriching-events/scopes/scope-synchronization/flutter.mdx

```diff
@@ -0,0 +1,12 @@
+### Scope Synchronization
```

Comment:
@denrase The page that uses this file includes needs to add a supported platform otherwise it's not visible, See https://sentry-docs-git-fork-denrase-denrase-document-flutter-sy-b4cbd9.sentry.dev/platforms/flutter/enriching-events/scopes/

Reply:
Pushed a change